default.vim

```diff
@@ -0,0 +1,85 @@
+" ~/Odoo-Development/oca/stock-logistics-workflow/default.vim:
```

Comment:
This file has to be removed :)

Reply:
The file is still there

Reply:
Mhm, for some reason deleting the file and commiting with --amend option does not work. Will remove it and then squash the commit.

Reply:
@janikvonrotz still not ;) Use `git rm --cached default.vim` to remove the file from the git repo without removing it from your filesystem if it is what you want.

Reply:
@sebalix Using vim to run git rebase interactively seems to be the issue...